apps/web/components/v2/eventtype/AvailabilityTab.tsx

```diff
@@ -0,0 +1,146 @@
+import { FormValues } from "pages/v2/event-types/[type]";
+import { Controller, useFormContext } from "react-hook-form";
+
+import dayjs from "@calcom/dayjs";
+import classNames from "@calcom/lib/classNames";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { weekdayNames } from "@calcom/lib/weekday";
+import { trpc } from "@calcom/trpc/react";
+import { Icon } from "@calcom/ui";
+import { SkeletonText } from "@calcom/ui/v2";
+import Button from "@calcom/ui/v2/core/Button";
+import Select from "@calcom/ui/v2/core/form/Select";
+
+import { QueryCell } from "@lib/QueryCell";
+
+import { AvailabilitySelectSkeletonLoader } from "@components/availability/SkeletonLoader";
+
+type AvailabilityOption = {
+  label: string;
+  value: number;
+};
+
+const AvailabilitySelect = ({
+  className = "",
+  ...props
+}: {
+  className?: string;
+  name: string;
+  value: number;
+  onBlur: () => void;
+  onChange: (value: AvailabilityOption | null) => void;
+}) => {
+  const query = trpc.useQuery(["viewer.availability.list"]);
+
+  return (
+    <QueryCell
```

Comment:
NIT: To my understanding we are trying to move away from Query Cell? @zomars

Reply:
is this a blocker? @zomars @emrysal @sean-brydon 

if not lets merge

Reply:
I don't believe so @PeerRich  - We still use it across the app. New issue should be created to remove query cell if we are planning to not use it anymore however